src/ParseServer.js

```diff
@@ -214,8 +218,39 @@ class ParseServer {
     return this._app;
   }
 
-  handleShutdown() {
+  /**
+   * Stops the parse server, cancels any ongoing requests and closes all connections.
+   *
+   * Currently, express doesn't shut down immediately after receiving SIGINT/SIGTERM
+   * if it has client connections that haven't timed out.
+   * (This is a known issue with node - https://github.com/nodejs/node/issues/2642)
+   *
+   * @returns {Promise<void>} a promise that resolves when the server is stopped
+   */
+  async handleShutdown() {
```

Comment:
Is this a private or public API? If public, it'd look like a breaking change becoming async?

Reply:
It's publicly available but not documented. It returned a promise before so I don't think it's breaking.